src/keboola_mcp_server/clients/data_science.py

```diff
@@ -0,0 +1,252 @@
+        validation_alias=AliasChoices('lastRequestTimestamp', 'last_request_timestamp'),
+        default=None,
+        description='The last request timestamp',
+    )
+    last_start_timestamp: Optional[str] = Field(
+        validation_alias=AliasChoices('lastStartTimestamp', 'last_start_timestamp'),
+        default=None,
+        description='The last start timestamp',
+    )
+    url: Optional[str] = Field(
+        validation_alias=AliasChoices('url', 'url'), description='The URL of the running data app', default=None
+    )
+    auto_suspend_after_seconds: int = Field(
+        validation_alias=AliasChoices('autoSuspendAfterSeconds', 'auto_suspend_after_seconds'),
+        description='The auto suspend after seconds',
+    )
+    size: Optional[str] = Field(
+        validation_alias=AliasChoices('size', 'size'), description='The size of the data app', default=None
+    )
+
+
+class DataAppConfig(BaseModel):
+    """
+    The simplified data app config model, which is used for creating a data app within the mcp server.
+    """
+
+    class Parameters(BaseModel):
+        class DataApp(BaseModel):
+            slug: str = Field(description='The slug of the data app')
+            streamlit: dict[str, str] = Field(description='The streamlit config.toml file')
+            secrets: Optional[dict[str, str]] = Field(description='The secrets of the data app', default=None)
+
+        size: str = Field(description='The size of the data app')
+        auto_suspend_after_seconds: int = Field(
+            validation_alias=AliasChoices('autoSuspendAfterSeconds', 'auto_suspend_after_seconds'),
+            serialization_alias='autoSuspendAfterSeconds',
+            description='The auto suspend after seconds',
+        )
+        data_app: DataApp = Field(
+            description='The data app sub config',
+            serialization_alias='dataApp',
+            validation_alias=AliasChoices('dataApp', 'data_app'),
+        )
+        id: Optional[str] = Field(description='The id of the data app', default=None)
+        script: Optional[list[str]] = Field(description='The script of the data app', default=None)
+        packages: Optional[list[str]] = Field(
+            description='The python packages needed to be installed in the data app', default=None
+        )
+
+    class Authorization(BaseModel):
+        class AppProxy(BaseModel):
+            auth_providers: list[dict[str, Any]] = Field(description='The auth providers')
+            auth_rules: list[dict[str, Any]] = Field(description='The auth rules')
+
+        app_proxy: AppProxy = Field(description='The app proxy')
+
+    parameters: Parameters = Field(description='The parameters of the data app')
+    authorization: Authorization = Field(description='The authorization of the data app')
+    storage: dict[str, Any] = Field(description='The storage of the data app', default_factory=dict)
+
+
+class AsyncDataScienceClient(KeboolaServiceClient):
```

Comment:
you can drop `Async` since all our clients are async now; same for other clients